src/Model/Post.php

```diff
@@ -406,49 +407,52 @@ protected function init() {
 
 					return false;
 				},
-				'toPing'          => function() {
+				'toPing'           => function() {
 					return ! empty( $this->data->to_ping ) && is_array( $this->data->to_ping ) ? implode( ',', (array) $this->data->to_ping ) : null;
 				},
-				'pinged'          => function() {
+				'pinged'           => function() {
 					return ! empty( $this->data->pinged ) && is_array( $this->data->pinged ) ? implode( ',', (array) $this->data->pinged ) : null;
 				},
-				'modified'        => function() {
+				'modified'         => function() {
 					return ! empty( $this->data->post_modified ) && '0000-00-00 00:00:00' !== $this->data->post_modified ? $this->data->post_modified : null;
 				},
-				'modifiedGmt'     => function() {
+				'modifiedGmt'      => function() {
 					return ! empty( $this->data->post_modified_gmt ) ? Types::prepare_date_response( $this->data->post_modified_gmt ) : null;
 				},
-				'parentId'        => function() {
+				'parentId'         => function() {
+					return ( ! empty( $this->data->post_type ) && ! empty( $this->data->post_parent ) ) ? Relay::toGlobalId( $this->data->post_type, $this->data->post_parent ) : null;
```

Comment:
Hmmm 🤔 so this is going to cause a breaking change, since consumers would be expecting the numeric DB ID from this field previously. I think it's OK to make this breaking change if that's the direction we're going to take with:  https://github.com/wp-graphql/wp-graphql/issues/998.